colors.go

```diff
@@ -199,12 +205,12 @@ func (sm styleMap) parseFile(path string) {
 			key = filepath.Clean(key)
 		}
 
-		sm[key] = applyAnsiCodes(val, tcell.StyleDefault)
+		sm.styles[key] = applyAnsiCodes(val, tcell.StyleDefault)
```

Comment:
Shouldn't parsing the colors file also check for `ln target`? The code here should be similar to `icons.go`.

Reply:
Good call, I forgot about color file parsing.